eventstore/store.go

```diff
@@ -213,6 +215,11 @@ func (s *Store) Register(name string, defaultInstance interface{}) (*Model, erro
 	s.lock.Lock()
 	defer s.lock.Unlock()
 
+	diType := reflect.TypeOf(defaultInstance)
```

Comment:
Check that the registered default instance of the model has a correct type.
They should always be pointers to structs. e.g. `m.Register("Person", &Person{})`.